code/modules/mob/living/simple_animal/guardian/types/ranged.dm

```diff
@@ -131,8 +132,3 @@
 	// To stop scout mode from moving when recalled
 	incorporeal_move = FALSE
 	. = ..()
-
-/mob/living/simple_animal/hostile/guardian/ranged/AttackingTarget()
```

Comment:
Also what's this.

Reply:
I have no idea what happened there. I'll revert that.